modules/Bio/EnsEMBL/Compara/RunnableDB/ReindexMembers/DeletionHealthcheck.pm

```diff
@@ -0,0 +1,97 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::ReindexMembers::DeletionHealthcheck
+
+=head1 DESCRIPTION
+
+This Runnable checks if the number of gene trees or homologies decreased significantly 
+compared to the previous pipeline database. 
+The difference between the current and previous counts is normalised by the previous count
+and expressed as a perecentage.
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::ReindexMembers::DeletionHealthcheck;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+
+sub param_defaults {
+
+}
+
+
+sub fetch_input {
+    my $self = shift @_;
+
+    $self->param('curr_db', $self);
+
+}
+
+sub run {
+    my $self = shift @_;
+    my $prev_dbc = $self->get_cached_compara_dba('prev_tree_db')->dbc;
+    my $curr_dbc = $self->param('curr_db')->dbc;
```

Comment:
If accessing the current pipeline database, the `compara_dba` method could be used instead.
```suggestion
    my $curr_dbc = $self->compara_dba->dbc;
```